src/components/Button.js

```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import '../App.css';
 
-function Button(props) {
-  const { name, color, wide } = props;
+const Button = (props) => {
+  const {
+    name, color, wide, clickHandler,
+  } = props;
   const width = wide ? '50%' : '25%';
   const style = { backgroundColor: color, width };
-  return <div className="Button" style={style}>{name}</div>;
-}
+
+  const handleClick = () => clickHandler();
+
+  /* eslint-disable jsx-a11y/click-events-have-key-events */
+  return <div role="button" tabIndex={name} className="Button" style={style} onClick={() => handleClick()}>{name}</div>;
```

Comment:
You only need to pass the function `handleClick` as the value of `onClick={handleClick}`